micro-ui/web/micro-ui-internals/packages/css/CHANGELOG.md

```
@@ -1,5 +1,8 @@
 # Changelog
+## [1.8.41] - [26-Aug-2025]
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)